db/data_migration/20150209134053_remove_self_assessment_detailed_guide_category.rb

```diff
@@ -0,0 +1,33 @@
+require 'gds_api/router'
+router_api = GdsApi::Router.new(Plek.current.find('router-api'))
+
+REDIRECT_TO = "/government/organisations/hm-revenue-customs"
+
+categories = MainstreamCategory.where(parent_tag: "tax/self-assessment")
+
+categories.each do |category|
+  old_path = Whitehall.url_maker.mainstream_category_path(category)
+  puts "removing category: #{category.title}"
+  guides = category.detailed_guides
+
+  puts "\t registering redirect: #{old_path} => #{REDIRECT_TO}"
+  router_api.add_redirect_route(old_path, 'exact', REDIRECT_TO)
+
+  puts "\t removing association to category from #{guides.count} guides"
+  guides.each do |guide|
+    if guide.primary_mainstream_category_id == category.id
```

Comment:
It would be cleaner if the code for dissociating a category from the guide was a method on the guide itself, but that doesn't have to block this PR.